runtime/lua/vim/_defaults.lua

```diff
@@ -925,6 +925,25 @@ do
       end
     end
   end
+
+  vim.api.nvim_create_autocmd('VimEnter', {
+    group = vim.api.nvim_create_augroup('nvim.find_exrc', {}),
+    desc = 'Find project-local configuration',
+    callback = function()
+      if vim.o.exrc then
+        local paths = vim.fs.find(
+          { '.nvim.lua', '.nvimrc', '.exrc' },
+          { type = 'file', upward = true }
```

Comment:
`.nvimrc` and `.exrc` aren't Lua files, so loadstring won't work with these.

I think it's fine to only support this for `.nvim.lua`, at least for now. As long as that caveat is well-documented.

Reply:
Ah of course. I think it's fine too, but it wouldn't add much complexity to find out which filename matched, and use the corresponding eval function for that type (lua/vimscript).

Reply:
Agreed, sounds good.